### PR TITLE
Configure testing pipeline in Jenkins

### PR DESCRIPTION
**Pull request by adrianmo**

### CODE REVIEW

File: ui/run.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ fi
 run() {
     run="${1}"
     shift
-    sudo docker run --rm -it --privileged --net=host ${default_mount_opts[@]} ${image_release} ${run} ${@}
+    sudo docker run --rm -i --privileged --net=host ${default_mount_opts[@]} ${image_release} ${run} ${@}
```
Comment (adrianmo) [inline ui/run.sh]: I needed to change this for the Ansible playbook to work.

**Comment by adrianmo** [comment]
@padthaitofuhot Can you take a first look at it?

File: ui/run.sh (hunk 1 above)
Comment (padthaitofuhot) [inline ui/run.sh]: Please see #298 first.

Comment (padthaitofuhot) [inline ui/run.sh]: Not once #298 is merged... So please revert this commit and let's merge #298 and rebase this PR on top of that.

Comment (adrianmo) [inline ui/run.sh]: PR rebased and `run.sh` left untouched.

**Comment by adrianmo** [comment]
@padthaitofuhot this is ready for review.

**Comment by adrianmo** [comment]
@padthaitofuhot thanks! you are right about the `extraVars`. And also yes, I agree with the `tf_to_hosts` script, I will look into how to refactor it and include it in the inventory script in a different PR.

**Pull request merged**